src/server/on-off-farm/index.js

```diff
@@ -0,0 +1,36 @@
+import {
+  onOffFarmGetController,
+  onOffFarmPostController
+} from './controller.js'
+
+/**
+ * Sets up the routes used in the home page.
+ * These routes are registered in src/server/router.js.
+ */
+
+/**
+ * @satisfies {ServerRegisterPluginObject<void>}
+ */
+export const onOffFarm = {
+  plugin: {
+    name: 'to-or-from-own-premises',
+    register(server) {
+      server.route([
+        {
+          method: 'GET',
+          path: '/to-or-from-own-premises',
+          ...onOffFarmGetController
```

Comment:
I'm sure we'll figure this out later, but it feels weird to me that we're expecting some subset of properties on the route to be defined here, and some in the control itself.  Leaves open ambiguity for where a route option would be defined. 

It'd feel more natural to have the definition file be:

```javascript
const onOffFarmGetHandler = (req, h) => {
  //...
}
```

And then have this file: 
```javascript
server.route([
  {
    method: 'GET',
    path: '/to-or-from-own-premises',
    handler: onOffFarmGetHandler
}
```

This is a comment to get your thoughts rather than a request for change.